jellyfin_kodi/downloader.py

```diff
@@ -129,8 +129,7 @@ def get_filtered_section(parent_id=None, media=None, limit=None, recursive=None,
 
 def get_movies_by_boxset(boxset_id):
 
-    for items in get_items(boxset_id, "Movie"):
-        yield items
```

Comment:
We currently support kodi 17 and 18 which both use Python 2.7, this syntax was introduced in python 3 and so we can't use it

Reply:
Ahh, haven't written any python-2.* code in ages. I believe the yield commits might be the only ones that are python3 specific since the other commits are for boolean & if statement/expression changes.

Reply:
Reverted this commit.